src/main/java/frc/utils/loggerutils/LoggerUtils.java

```diff
@@ -47,7 +47,7 @@ public static void startReplayLogger() {
     }
 
     private static void startNonReplayLogger(LogSaveSpot logSaveSpot) {
-        setLoggingPath(logSaveSpot.savePath);
+        setLoggingPath(logSaveSpot.savePath.toString());
```

Comment:
Why does setLoggingPath get a String as a parameter and not a Path?

Reply:
Because if i give a path i would need to do toString twice or save var with the string, and its duplicate code

Reply:
But the name suggests it gets a path. If you don't want to turn every path from a String to a Path I see no reason to make some paths different

Reply:
I am not sure what i prefer, but the functions in gets string so what do you think?

Reply:
I think that if everything gets a string and not a path there is no reason to change the Constants into paths

Reply:
First of all it's a better approach to handle paths as paths, like u use enums for stuff and not just 1 2 3 4.
The second thing is the line in the image is very long so i need it to be shorter

Reply:
I know but I think if you are using Paths you should go all the way and not just put a .to string() in the middle of the process.
Could you imagine if I put angles in a rotation2d but then I converted it into a double in the middle of my code and used it like that in my functions?

Reply:
look on the code again

Reply:
i changed it